org.eclipse.scanning.example/src/org/eclipse/scanning/example/detector/MandelbrotDetector.java

```diff
@@ -152,7 +153,7 @@ private NXdetector createNexusObject(NexusScanInfo info) throws NexusException {
 		if (model.isSaveSpectrum()) detector.setDataset(FIELD_NAME_SPECTRUM_AXIS, DatasetFactory.createLinearSpace(0.0, model.getMaxRealCoordinate(), model.getPoints(), Dataset.FLOAT64));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "createLinearSpace"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)